NebulaPatcher/Patches/Dynamic/GameData_Patch.cs

```diff
@@ -296,6 +298,19 @@ public static void SetForNewGame_Postfix(GameData __instance)
         }
     }
 
+    [HarmonyPostfix, HarmonyPriority(Priority.High)]
+    [HarmonyPatch(nameof(GameData.NewGame))]
+    public static void NewGame_Postfix(GameData __instance)
+    {
+        if (!Multiplayer.IsActive || Multiplayer.Session.LocalPlayer.IsHost)
+        {
+            return;
+        }
```

Comment:
Minor nitpick, could be simplified to:
```suggestion
        if (!Multiplayer.IsActive || Multiplayer.Session.LocalPlayer.IsHost) return;
```